tests/multigrid/transfer_matrix_free_13.cc

```diff
@@ -69,6 +69,19 @@ check()
   user_constraints.close();
   mg_constrained_dofs.add_user_constraints(0, user_constraints);
 
+  {
+    // test clear_user_constraints()
+    AffineConstraints<double> tmp_user_constraints;
+    tmp_user_constraints.reinit(relevant_dofs);
+    tmp_user_constraints.add_line(face_dofs[0]);
+    tmp_user_constraints.set_inhomogeneity(face_dofs[0], 100.);
+    tmp_user_constraints.close();
+
+    mg_constrained_dofs.add_user_constraints(0, tmp_user_constraints);
+    mg_constrained_dofs.clear_user_constraints();
```

Comment:
Is the point here that it would fail without the call to clear ()? I would think calling clear() and then checking that the constraints are empty would be easier.

Reply:
Yes, it will fail without calling clear_user_constraints().
OK, I will change it.